R/two-column-layouts.R

```diff
@@ -214,3 +241,41 @@ gt_two_column_layout <- function(tables = NULL, output = "viewer",
     double_tables
   }
 }
+
+extract_tab_header_and_style <- function(table) {
+  raw_html <- gt::as_raw_html(table, inline_css = FALSE) %>%
+    xml2::read_html()
+  
+  gt_title <- raw_html %>%
+    xml2::xml_find_first("//*[contains(concat(' ',normalize-space(@class),' '),' gt_title ')]")
+  
+  gt_subtitle <- raw_html %>%
+    xml2::xml_find_first("//*[contains(concat(' ',normalize-space(@class),' '),' gt_subtitle ')]")
+  
+  gt_table_id <- raw_html %>%
+    xml2::xml_find_all("//body/div") %>%
+    xml2::xml_attr("id")
+  
+  s <- raw_html %>%
+    xml2::xml_find_first("//style") %>%
+    xml2::xml_contents() %>%
+    xml2::xml_text() %>%
+    gsub(gt_table_id, "mycombinedtable", x = .) %>%
+    gsub("mycombinedtable table", "mycombinedtable div", x = .)
```

Comment:
The table selector sets font. Since we put the header in div tags we replace table with div in the style sheet.